wot-wg-2023-draft.html

```diff
@@ -193,18 +194,29 @@ <h1 id="title">PROPOSED Web of Things Working Group Charter</h1>
         WoT IG <a href="https://w3c.github.io/wot/current-practices/wot-practices.html#plugfests">PlugFests</a>
         during development.
         Open-source implementations will be prioritized.
+-->
       </p>
     </div>
 
     <section id="scope" class="scope">
       <h2>Scope</h2>
       <!-- What exactly is in scope, and out of scope.
           For legal review. Be brief. -->
-      <p>The Working Group has the following work items in scope:</p>
+      <p>During this charter, the Working Group would:</p>
+      <ul>
+      <li>Update the Architecture, Thing Description, Profile, and Discovery specifications
+      to address major issues discovered during implementation of the 1.0 and 1.1 specifications.
+      Unlike the last charter, these would not be limited to backward-compatible updates.</li> 
+      <li>Reorganize Security Schemes and Protocol/Payload Bindings to be more modular and less specific
+      to HTTP.</li>
```

Comment:
This reads as though protocol/payload binding (templates) are currently specific to HTTP